snooty/diagnostics.py

```diff
@@ -340,6 +340,57 @@ def __init__(
         self.entry = entry
 
 
+class UnknownTabset(Diagnostic):
+    severity = Diagnostic.Level.error
+
+    def __init__(
+        self,
+        tabset: str,
+        start: Union[int, Tuple[int, int]],
+        end: Union[None, int, Tuple[int, int]] = None,
+    ) -> None:
+        super().__init__(
+            f"tabset {tabset} is not defined in rstspec.toml", start, end,
```

Comment:
Start upper case, and wrap values in double-quotes. You'll have to swap the string quotes for triple quotes, e.g. `f"""Tabset "{tabset} is not...`